flink-core/src/test/java/org/apache/flink/api/common/ExecutionConfigTest.java

```diff
@@ -159,4 +160,21 @@ public void testExecutionConfigSerialization() throws IOException, ClassNotFound
 		assertEquals(sysoutLoggingEnabled, copy1.isSysoutLoggingEnabled());
 		assertEquals(parallelism, copy1.getParallelism());
 	}
+
+	@Test
+	public void testGlobalParameters() {
+		final ExecutionConfig config = new ExecutionConfig();
+
```

Comment:
nit: maybe we get `GlobalJobParameters empty = config.getGlobalJobParameters()` first and then do the assertion.